mlflow/store/tracking/utils/sql_trace_metrics_utils.py

```diff
@@ -75,10 +76,18 @@ class TraceMetricsConfig(TypedDict):
     },
 }
 
-# TODO: add spans and assessments metrics configs
+ASSESSMENTS_METRICS_CONFIGS: dict[str, TraceMetricsConfig] = {
+    "assessment": {
```

Comment:
```suggestion
    "assessment_count": {
```
I'll convert this to a constant as well after previous PR is merged